src/train.py

```
@@ -175,22 +199,78 @@ def train_model(args: argparse.Namespace) -> int:
         try:
             from ultralytics import YOLO
-        except ImportError:
-            logger.error("ultralyticsパッケージがインストールされていません")
+        except ImportError as e:
+            logger.error(f"ultralyticsパッケージのインポートエラー: {e}")
+            logger.error(f"Python version: {sys.version}")
+            logger.error(f"Python path: {sys.path}")
+            try:
+                import ultralytics
+                logger.error(f"ultralytics module found at: {ultralytics.__file__}")
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)